haproxy/recipes/ctl.rb

```diff
@@ -18,3 +18,28 @@
 link '/etc/init.d/haproxyctl' do
   to "#{base_path}/haproxyctl/bin/haproxyctl"
 end
+
+### haproxyctl statsd stuff ###
+statsd_host = node['haproxy']['ctl']['statsd']['host']
+statsd_port = node['haproxy']['ctl']['statsd']['port']
+directory '/etc/haproxy/haproxyctl' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  recursive true
+  action :create
+end
+
+template '/etc/haproxy/haproxyctl/instance-name' do
+  source 'haproxyctl.instance-name.erb'
+  variables(
+    :stack_name => node['easybib_deploy']['envtype'],
+    :host_name  => ::EasyBib.get_hostname(node, true)
```

Comment:
I would also avoid this, and include the lib before-hand.

Here is how you include (or maybe monkey-patch):
https://github.com/till/easybib-cookbooks/blob/master/statsd/recipes/default.rb#L1

Then you can use a method directly:
https://github.com/till/easybib-cookbooks/blob/master/statsd/recipes/default.rb#L30

`is_aws` vs. `::EasyBib.is_aws`